README.md

```diff
@@ -26,20 +26,42 @@ Variables:
 Dependencies:
 
 * Vagrant
-* Virtualbox
+* Docker or VirtualBox
 * virtualization enabled in bios
 
 Procedure:
 
 1. Clone the pelux-manifests git repository.
 2. Start vagrant
 ```bash
-MANIFEST=<manifest> BITBAKE_IMAGE=<image> vagrant up
+vagrant up --provider="docker"
+```
+3. Set variables to be used below
+```bash
+bitbake_image="core-image-pelux"
+yoctoDir="/home/vagrant/pelux_yocto"
+```
+4. Setup bitbake with correct local.conf and bblayers.conf
+```bash
+vagrant ssh -c \"TEMPLATECONF=${yoctoDir}/sources/meta-pelux-bsp-intel/conf
+                 /vagrant/vagrant-cookbook/yocto/fetch-sources-for-recipes.sh
+                 ${yoctoDir}
+                 ${bitbake_image}
+               \"
+```
+5. Bitbake the PELUX image
+```bash
+vagrant ssh -c \"/vagrant/vagrant-cookbook/yocto/build-images.sh
+                 ${yoctoDir}
+                 ${bitbake_image}
+               \"
+```
+6. Move the built images to the host
+```bash
+vagrant ssh -c "cp -r ${yoctoDir}/build/tmp/deploy/images /vagrant"
 ```
 
-The virtual machine started via vagrant will sync the cloned git repository and use the manifests contained in it
-to set up the build environment. This means that the branch/commit currently checked out will determine what version
-is being built.
+The container/virtual machine started via vagrant will sync the cloned git repository and use the manifests contained in it to set up the build environment. This means that the branch/commit currently checked out will determine what version is being built. The final step will copy the image directory containing the built images to the directory on the host where vagrant was started.
```

Comment:
I don't know if anyone thought about the pros and cons of breaking lines. But in most of our markdown docs, we manually insert linebreaks. I think we should be consistent with that whatever way we choose to do it.